### PR TITLE
Add the ability to link to individual part of the code of conduct

### PR DESCRIPTION
**Pull request by StrangeWill**

### CODE REVIEW

**Comment by StrangeWill** [comment]
Example: https://deploy-preview-207--devanooga.netlify.com/code-of-conduct/#condict-offensive-language

File: _pages/coc.md [hunk 1]
```diff
@@ -12,12 +13,12 @@ Follow the golden rule: treat others in the way that you would want to be treate
...
-- Come prepared to share ideas _and defend them_. We want a place that is fun and thought provoking.
-- You may see ideas that are contrary to yours and that is ok.
-- While it is ok to express contrary ideas, it is **NOT** ok to attack, defame, or otherwise harass another member. It **IS** ok (and encouraged) to criticize ideas that you disagree with.
-- Bring a positive attitude to all interactions.
-- Assume that everyone else is being light-hearted and acting in good faith unless demonstrated otherwise.
+- {:#condict-offensive-language} Avoid language that is unnecessarily offensive to other users
+- {:#condict-defend} Come prepared to share ideas _and defend them_. We want a place that is fun and thought provoking.
+- {:#conduct-tolerance-of-ideas} You may see ideas that are contrary to yours and that is ok.
+- {:#condict-attack-defame-harass} While it is ok to express contrary ideas, it is **NOT** ok to attack, defame, or otherwise harass another member. It **IS** ok (and encouraged) to criticize ideas that you disagree with.
+- {:#condict-positive} Bring a positive attitude to all interactions.
```
Comment (agarzola) [inline _pages/coc.md]: `condict` should be `conduct` here, as well as lines 16, 17, and 19.

File: _sass/_code-of-conduct.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.code-of-conduct {
+  li {
+    padding: 2px 5px;
+
+    &:target {
+
+      background-color: #ae81ff;
```
Comment (agarzola) [inline _sass/_code-of-conduct.scss]: This color doesn’t work well, in my opinion, with other colors used in the site. I suggest `#1E5594`.

File: _pages/coc.md (hunk 1 above)
Comment (StrangeWill) [reply _pages/coc.md]: I'm good at copying mistakes, thanks. 👍

File: _sass/_code-of-conduct.scss (hunk 2 above)
Comment (StrangeWill) [reply _sass/_code-of-conduct.scss]: Seems a little dark of a background, opinions?

Comment (agarzola) [reply _sass/_code-of-conduct.scss]: BTW, I arrived at the color above by checking the colors we use for blog tags against [the WebAIM Color Contrast Checker API](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=5F9ADE) and finding that they fail every test. 😭 

So if you like my suggestion, perhaps you’d like to roll that update into this PR? If you _don’t_ like my suggestion, try playing around with the contrast checker for a color that meets accessibility best practices, and use that color for both uses.

Comment (agarzola) [reply _sass/_code-of-conduct.scss]: @StrangeWill Oh! Sorry… I meant to also say: Add `color: #ffffff;`. 😄

Comment (StrangeWill) [reply _sass/_code-of-conduct.scss]: Ehhh trying the lilac color we use for the writer tags. Opinions?

Comment (agarzola) [reply _sass/_code-of-conduct.scss]: I don’t like it _aesthetically,_ but [the checker likes it](https://webaim.org/resources/contrastchecker/?fcolor=232323&bcolor=A97FCE).

Comment (StrangeWill) [reply _sass/_code-of-conduct.scss]: Hmm, I'll ponder this a bit more, I kind of felt the same.

**Pull request merged**